test/libsolidity/syntaxTests/tupleAssignments/empty_tuple_assignment.sol

```diff
@@ -0,0 +1,12 @@
+function f() {
```

Comment:
```suggestion
function f() public pure {
```

Reply:
I only added `pure`. Can't add `public` to free functions.

Reply:
Oh yeah, sorry about that, i just copied the text from an open file I had at the time.